common/tools/eslint-plugin-azure-sdk/src/rules/ts-no-window.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @file Rule to forbid usage of window.
+ * @author Maor Leger
+ */
+
+import { Rule } from "eslint";
+import { getRuleMetaData } from "../utils";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+export = {
+  meta: getRuleMetaData("ts-no-window", "forbid usage of window", "code"),
+  create: (context: Rule.RuleContext): Rule.RuleListener =>
+    ({
+      // callback functions
+
```

Comment:
I do not understand the purpose of this comment

Reply:
Fixed!